metabase/metabase
# Add timeline permissions

Epic https://github.com/metabase/metabase/issues/20289

Implements correct permission checks based on timeline's `collection`. 

How to test:
- Sign in as admin and create a timeline with events both in via collections and questions
- Create a readonly user and sign in
- Make sure it's possible to user timeline events, but not edit/archive them
